crates/astria-conductor/src/conductor.rs

```diff
@@ -167,12 +302,13 @@ impl Conductor {
         })
     }
 
-    /// Runs [`Conductor`] until it receives an exit signal.
+    /// Runs [`InnerConductorTask`] until it receives an exit signal.
     ///
     /// # Panics
     /// Panics if it could not install a signal handler.
-    async fn run_until_stopped(mut self) {
+    async fn run_until_stopped(mut self) -> eyre::Result<RestartToken> {
```

Comment:
I have it returning an error because otherwise if one of the conductor tasks fail it just shuts down and doesn't propagate the error. It does emit an error event, so I wouldn't call it totally silent, but it seemed like it didn't follow the same error-handling approach from the rest of the codebase. Happy to remove this if needed, though.